sdk/inc/azure/iot/az_iot_adu_client.h

```diff
@@ -119,7 +119,7 @@ typedef struct
    * The version for the update.
    */
   az_span version;
-} az_iot_adu_client_update_id;
+} az_iot_adu_update_id;
```

Comment:
Should we really drop the `_client`?

Reply:
We should, because this is to represent the update id in the update manifest. It's not specific to the client.

Reply:
already precedent with the device information